### ISSUE TITLE
UseTemplate() Layer z-index

### ISSUE THREAD

**Issue by sergiupetrita** [mpdf/mpdf#139]
Hello,

Is there a way to import a PDF in the document created with mPDF and set the background color.
Here https://mpdf.github.io/reference/mpdf-functions/usetemplate.html is a comment saying:

> The template will be printed onto the page as the bottom 'layer' i.e. anything else written to that page by mPDF will be written on top of thie template

Is there a way to have a page background color when importing a PDF?
I tried to use BeginLayer, but with no success.

```
$this->mpdf->BeginLayer(100);
$this->mpdf->SetImportUse();
$pagecount = $this->mpdf->SetSourceFile($contentPath);
for ($i=1; $i<=$pagecount; $i++) {
    $import_page = $this->mpdf->ImportPage($i);
    $this->mpdf->UseTemplate($import_page);
}
$this->mpdf->EndLayer();
```
 
Please help me out!
Thanks!

**Comment by finwe** [comment]
No, AFAIK that is not possible - I believe this is somehow clear from the description in the manual. Sorry.

Leaving open as possible enhancement fot the future.

**Comment by Ixreb** [comment]
Any solution for this? Thank you
